components/api/src/main/java/com/hotels/styx/api/ByteStream.java

```diff
@@ -50,6 +51,17 @@ public ByteStream(Publisher<Buffer> stream) {
         this.stream = requireNonNull(stream);
     }
 
+    /**
+     * Creates a new {@code ByteStream} from String.
+     *
+     * @param content ByteStream source
+     * @param charset Character set encoding
+     * @return ByteStream
+     */
+    public static ByteStream from(String content, Charset charset) {
```

Comment:
Do you think we should have a factory to construct the object from the String instead of from a byte source?

Reply:
Yes I think we should. I'll add this.